src/views/darkroom.c

```diff
@@ -1499,7 +1499,7 @@ static int _iso_12646_get_border(dt_develop_t *d)
 {
   if(d->iso_12646.enabled)
   {
-    return MIN(1.75 * darktable.gui->dpi, 0.3 * MIN(d->width, d->height));
```

Comment:
I think we should make those magic numbers entries in darktablerc.xml.in. Not visible on the prefs, right but at least one could change the size of the border without recompiling dt. Also having those numbers in the darktablerc will somehow document them. Can you do that?